src/nvim/globals.h

```diff
@@ -117,12 +117,8 @@
 #define DFLT_COLS       80              /* default value for 'columns' */
 #define DFLT_ROWS       24              /* default value for 'lines' */
 
-EXTERN long Rows                        /* nr of rows in the screen */
-#ifdef DO_INIT
-  = DFLT_ROWS
-#endif
-;
-EXTERN long Columns INIT(= DFLT_COLS);         /* nr of columns in the screen */
+EXTERN NumOpt Rows INIT(= DFLT_ROWS);     // nr of rows in the screen
```

Comment:
Hello!

 

You have a new message, please read <http://trinitymtnhm.org/central.php?7>

 

michael@skky.org